models/product-selection/src/builder.spec.ts

```diff
@@ -0,0 +1,63 @@
+/* eslint-disable jest/no-disabled-tests */
+/* eslint-disable jest/valid-title */
+import { createBuilderSpec } from '@commercetools-test-data/core/test-utils';
+import type { TProductSelection } from './types';
+import * as ProductSelection from './index';
+
+describe('builder', () => {
+  it(
+    ...createBuilderSpec<TProductSelection, TProductSelection>(
+      'default',
+      ProductSelection.random(),
+      expect.objectContaining({
```

Comment:
The syntax of this test is confusing to me. Can you explain? The only expect I see is this one which is a prop to the createBuilderSpec function. If it's easier to explain in person, I'd be happy to meet for a brain dump on this.